deps/chakrashim/src/jsrtisolateshim.cc

```diff
@@ -224,8 +224,7 @@ void IsolateShim::PushScope(
   this->contextScopeStack = scope;
 
   // Don't crash even if we fail to set the context
-  JsErrorCode errorCode = JsSetCurrentContext(contextShim->GetContextRef());
-  CHAKRA_ASSERT(errorCode == JsNoError);
+  JsSetCurrentContext(contextShim->GetContextRef());
```

Comment:
For debug build they are not ignored given that they are used in assert to make sure the value is `JsNoError`. We could have something like this
```c++
#ifdef DEBUG
JsErrorCode errorCode = 
#endif
JsSetCurrentContext(contextShim->GetContextRef());
CHAKRA_ASSERT(errorCode == JsNoError); // This line will be removed for release.
```